branchtesting/src/main/java/branchexample/App.java

```diff
@@ -10,7 +10,8 @@ public void aMethod( String[] args )
     		// TODO fix this too..
     		int j = 5 / i;
     	}
-    	System.out.println("IN AMETHOD"); 
+    	System.out.println("IN AMETHOD");
+    	System.out.println("New issue"); 
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this use of System.out or System.err by a logger. ([squid:S106](https://sonarcloud.io/organizations/brian-cipollone-sonarsource-gith/rules#rule_key=squid%3AS106))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.example.bcipollone%3Abranchtesting&pullRequest=12&issues=AWWIdROyXKENQn1SC-ip&open=AWWIdROyXKENQn1SC-ip)

[ISSUE_KEY]: AWWIdROyXKENQn1SC-ip